python_scripts/ensemble.py

```diff
@@ -907,35 +918,36 @@ def plot_decision_function(X, y, clf, sample_weight=None, fit=True, ax=None):
 
 # %% [markdown]
 # We can observe that in our grid-search, the largest `max_depth` together with
-# largest `n_estimators` lead to the best performance.
+# largest `n_estimators` led to the best performance.
 #
-# Gradient-boosting decision tree
+# ### Gradient-boosting decision tree
 #
-# In gradient-boosting, parameters tuning is a combination of several
+# For gradient-boosting, parameter tuning is a combination of several
 # parameters instead of setting one after the other each parameter. The
 # important parameters are `n_estimators`, `max_depth`, and `learning_rate`.
```

Comment:
This is not so clear. Have we tuned parameters one at a time previously? We tuned for a combination of parameters?

Reply:
My understanding is that in gradient-boosting, hyper parameters are strongly coupled, just as `max_depth` and `learning_rate`. 
But I guess that hyper parameters are usually always a bit coupled...
So, in my opinion
> instead of setting one after the other each parameter

seems to be a bit inappropriate, since we always use grid search.